File: 411/a.java
import java.util.Scanner;

public class a {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String pswd = in.nextLine();
        in.close();

        if (pswd.length() > 4 && ! sameCase(pswd) && containsDigit(pswd)) {
            System.out.println("Correct");
        } else {
            System.out.println("Too weak");
        }
    }

    private static boolean sameCase(final String pswd) {
        int up = 0, down = 0;

        for (int i = 0; i < pswd.length(); i++) {
            int code = (int) pswd.charAt(i);

            if (code > 96) {
                down++;
            } else if (code > 64 && code < 91) {
                up++;
            }
        }

        return up < 1 || down < 1;
    }

    private static boolean containsDigit(final String pswd) {
        for (int i = 0; i < pswd.length(); i++) {
            int code = (int) pswd.charAt(i);

            if (47 < code && code < 58) {
                return true;
            }
        }

        return false;
    }
}

File: 411/a.cpp
#include <iostream>
#include <string>

using namespace std;

inline bool has_digit(string &pswd) {
  for (int i = 0, n = pswd.length(); i < n; ++i) {
    if (pswd[i] >= '0' && pswd[i] <= '9') {
      return true;
    }
  }

  return false;
}

inline bool has_letter(string &pswd, bool small = false) {
  for (int i = 0, n = pswd.length(); i < n; ++i) {
    if (small) {
      if (pswd[i] >= 'a' && pswd[i] <= 'z') {
        return true;
      }
    } else {
      if (pswd[i] >= 'A' && pswd[i] <= 'Z') {
        return true;
      }
    }
  }

  return false;
}

inline bool check_pswd(string &pswd) {
  return pswd.length() >= 5u && has_letter(pswd) && has_letter(pswd, true) && has_digit(pswd);
}

int main() {
  string pswd; cin >> pswd;
  cout << (check_pswd(pswd) ? "Correct" : "Too weak") << endl;
  return 0;
}
